pkg/api/accesscontrol.go

```diff
@@ -434,10 +435,10 @@ func (hs *HTTPServer) declareFixedRoles() error {
 		Role: ac.RoleDTO{
 			Name:        "fixed:folders:creator",
 			DisplayName: "Creator",
-			Description: "Create folders.",
+			Description: "Create root level folders",
```

Comment:
```suggestion
			Description: "Create folders under assigned root level folders",
```

Since you are not able to create folders under the roor level